[quant] Use absolute import of the parent folder without alias.

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #24217 [quant] AddReLU module
* **#24792 [quant] Use absolute import of the parent folder without alias.**

This will prevent the circular dependencies in the future

Pull Request resolved: https://github.com/pytorch/pytorch/pull/24792

Differential Revision: [D16868861](https://our.internmc.facebook.com/intern/diff/D16868861)

Commit:
Use absolute import of the parent folder without alias.

This will prevent the circular dependencies in the future

zafartahirov commented:
main changes:
1. Use relative import for local files as it is very unlikely to have circular deps within the same folder
2. Use absolute import without aliasing for the rest.

Commit:
Update on "Use absolute import of the parent folder without alias."


This will prevent the circular dependencies in the future

Differential Revision: [D16868861](https://our.internmc.facebook.com/intern/diff/D16868861)

Commit:
Update on "Use absolute import of the parent folder without alias."


This will prevent the circular dependencies in the future

Differential Revision: [D16868861](https://our.internmc.facebook.com/intern/diff/D16868861)

jianyuh reviewed (approved):
LGTM! Thanks for the refactoring!

Commit:
Update on "Use absolute import of the parent folder without alias."


This will prevent the circular dependencies in the future

Differential Revision: [D16868861](https://our.internmc.facebook.com/intern/diff/D16868861)
